src/sequelize.ts

```diff
@@ -58,6 +54,33 @@ export function BigNumberStringType (propName: string): Partial<ModelAttributeCo
   }
 }
 
+/**
+ * consider that the field will be stored as string in the data base,
+ * so you not be able to use number comparision when querying
+ * BigNumberStringType for sequelize models
+ * @param propName
+ * @constructor
+ */
+export function BigNumberStringType (propName: string): Partial<ModelAttributeColumnOptions> {
+  return {
+    type: DataType.STRING(),
+    ...bigNumberGetterSetter(propName)
+  }
+}
+
+/**
+ * consider that the field will be stored as number in the data base,
+ * BigNumberStringType for sequelize models
```

Comment:
```suggestion
 * Consider that the field will be stored as BigInt in the database. 
 * Based on the database engine this might have limited precision. 
 * Using comparison and ordering is supported.
```